scanner/src/main/kotlin/Scanner.kt

```diff
@@ -67,13 +82,22 @@ fun scanOrtResult(
                 "result."
     }
 
-    // Add the projects as packages to scan.
+    // Determine the projects to scan as packages.
     val consolidatedProjects = Downloader.consolidateProjectPackagesByVcs(ortResult.getProjects(skipExcluded))
     val consolidatedReferencePackages = consolidatedProjects.keys.map { it.toCuratedPackage() }
 
-    val packagesToScan = (consolidatedReferencePackages + ortResult.getPackages(skipExcluded)).map { it.pkg }
-    val results = runBlocking { scanner.scanPackages(packagesToScan, outputDirectory, downloadDirectory) }
-    val resultContainers = results.map { (pkg, results) ->
+    val projectResults = runBlocking {
+        val packagesToScan = consolidatedReferencePackages.map { it.pkg }
+        projectScanner.scanPackages(packagesToScan, outputDirectory, downloadDirectory)
+    }
+
+    // Scan the packages from the ORT result.
+    val results = runBlocking {
+        val packagesToScan = ortResult.getPackages(skipExcluded).map { it.pkg }
+        scanner.scanPackages(packagesToScan, outputDirectory, downloadDirectory)
+    }
```

Comment:
This has a negative impact on performance, e.g. if there is a large project that takes a while to scan, existing scan results for packages will not be downloaded in the background anymore. You have to expand `runBlocking` to include both scanner calls, and I think you also need to use `async {}` here to make sure both scanners are processed in parallel, like it is used in `LocalScanner`. But then you also need to make the dispatchers used in `LocalScanner.scanPackages` static to not double the thread count.

Reply:
Done.

Reply:
Making the scan dispatcher static is not trivial as it currently uses the scanner name for the thread factory. Would you be fine with using a more generic name of "LocalScanner" here instead of the name of the real scanner?

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/2289/files#diff-b57870f9887a4464ed5851bf70e7fd71R81.

Reply:
Yes, that should be ok.